src/azure-cli/azure/cli/command_modules/signalr/_params.py

```diff
@@ -44,12 +45,14 @@ def load_arguments(self, _):
         c.argument('sku', help='The sku name of the signalr service. E.g. Standard_S1')
         c.argument('unit_count', help='The number of signalr service unit count', type=int)
         c.argument('service_mode', help='The service mode which signalr service will be working on', choices=SIGNALR_SERVICE_MODE_TYPE)
+        c.argument('enable_messaging_logs', help='The switch for messaging logs which signalr service will generate or not', choices=SIGNALR_DIAGNOSTIC_LOGS_SWITCH)
```

Comment:
Use `get_three_state_flag` instead of using `choices`, see https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-flags for details.